### DISCUSSION TITLE
Full saturation in RGB mode triggers restart

### DISCUSSION THREAD

**Discussion by boaschti** [arendst/Tasmota#19927] [Q&A]
### PROBLEM DESCRIPTION
If I configure a RGB led controller and set saturation to 100 percent in web gui then tasmota reboots. when I set the color after reboot it reboots again.

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [x] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x] Device used (e.g., Sonoff Basic): esp8266 12e + Leds and same behaviour on h801
- [x] Tasmota binary firmware version number used: Tasmota Version 13.2.0(sensors)-2_7_4_9(2023-10-19T09:02:00)
  - [x] Pre-compiled
  - [ ] Self-compiled
- [x] Flashing tools used: nodemcu flasher
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:
{"NAME":"Generic","GPIO":[1,1,1,1,1,1,1,1,1,1,1,1,1,1],"FLAG":0,"BASE":18}
{"Module":{"0":"Generic"}}
{"GPIO0":{"0":"None"},"GPIO1":{"0":"None"},"GPIO2":{"418":"PWM3"},"GPIO3":{"0":"None"},"GPIO4":{"225":"Relay2"},"GPIO5":{"224":"Relay1"},"GPIO9":{"0":"None"},"GPIO10":{"0":"None"},"GPIO12":{"417":"PWM2"},"GPIO13":{"419":"PWM4"},"GPIO14":{"226":"Relay3"},"GPIO15":{"416":"PWM1"},"GPIO16":{"227":"Relay4"},"GPIO17":{"0":"None"}}

```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:
{"Rule1":{"State":"OFF","Once":"OFF","StopOnError":"OFF","Length":0,"Free":511,"Rules":""}}
{"Rule2":{"State":"OFF","Once":"OFF","StopOnError":"OFF","Length":0,"Free":511,"Rules":""}}
{"Rule3":{"State":"OFF","Once":"OFF","StopOnError":"OFF","Length":0,"Free":511,"Rules":""}}

```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:
13:43:07.017 RSL: STATUS = {"Status":{"Module":0,"DeviceName":"Tasmota","FriendlyName":["Tasmota","Tasmota2","Tasmota3","Tasmota4",""],"Topic":"tasmota_1A8A13","ButtonTopic":"0","Power":31,"PowerOnState":3,"LedState":1,"LedMask":"FFFF","SaveData":1,"SaveState":1,"SwitchTopic":"0","SwitchMode":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"ButtonRetain":0,"SwitchRetain":0,"SensorRetain":0,"PowerRetain":0,"InfoRetain":0,"StateRetain":0,"StatusRetain":0}}
13:43:07.053 RSL: STATUS1 = {"StatusPRM":{"Baudrate":115200,"SerialConfig":"8N1","GroupTopic":"tasmotas","OtaUrl":"http://ota.tasmota.com/tasmota/release/tasmota-sensors.bin.gz","RestartReason":"Hardware Watchdog","Uptime":"0T00:09:50","StartupUTC":"2023-11-04T12:33:17","Sleep":50,"CfgHolder":4617,"BootCount":16,"BCResetTime":"2023-11-04T11:45:38","SaveCount":156,"SaveAddress":"F8000"}}
13:43:07.088 RSL: STATUS2 = {"StatusFWR":{"Version":"13.2.0(sensors)","BuildDateTime":"2023-10-19T09:02:00","Boot":5,"Core":"2_7_4_9","SDK":"2.2.2-dev(38a443e)","CpuFrequency":80,"Hardware":"ESP8266EX","CR":"379/699"}}
13:43:07.105 RSL: STATUS3 = {"StatusLOG":{"SerialLog":2,"WebLog":2,"MqttLog":0,"SysLog":0,"LogHost":"","LogPort":514,"SSId":["Servus nachad!",""],"TelePeriod":300,"Resolution":"558180C0","SetOption":["00008009","2805C80001000600003C5A0A192800000000","00000080","00006000","00004000","00000000"]}}
13:43:07.146 RSL: STATUS4 = {"StatusMEM":{"ProgramSize":701,"Free":300,"Heap":20,"ProgramFlashSize":4096,"FlashSize":4096,"FlashChipId":"1640E0","FlashFrequency":40,"FlashMode":"DOUT","Features":["00000809","8F9A8787","0415A005","B7F7BFCF","05DA9BC4","64367CC7","00084052","20000000","54000020","0008C081"],"Drivers":"1,2,3,4,5,6,7,8,9,10,12,14,16,17,20,21,24,29,34,62,65,66,68","Sensors":"1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,17,18,19,20,21,22,26,31,34,37,39,40,42,43,45,51,52,55,56,58,59,62,64,66,67,74,98,103,105","I2CDriver":"7,8,9,10,11,12,13,14,15,17,18,20,24,29,31,36,41,42,44,46,48,69,76"}}
13:43:07.194 RSL: STATUS5 = {"StatusNET":{"Hostname":"tasmota-1A8A13-2579","IPAddress":"192.168.178.69","Gateway":"192.168.178.1","Subnetmask":"255.255.255.0","DNSServer1":"192.168.178.1","DNSServer2":"0.0.0.0","Mac":"60:01:94:1A:8A:13","Webserver":2,"HTTP_API":1,"WifiConfig":4,"WifiPower":17.0}}
13:43:07.217 RSL: STATUS6 = {"StatusMQT":{"MqttHost":"","MqttPort":1883,"MqttClientMask":"DVES_%06X","MqttClient":"DVES_1A8A13","MqttUser":"DVES_USER","MqttCount":0,"MAX_PACKET_SIZE":1200,"KEEPALIVE":30,"SOCKET_TIMEOUT":4}}
13:43:07.239 RSL: STATUS7 = {"StatusTIM":{"UTC":"2023-11-04T12:43:07","Local":"2023-11-04T13:43:07","StartDST":"2023-03-26T02:00:00","EndDST":"2023-10-29T03:00:00","Timezone":"+01:00","Sunrise":"07:41","Sunset":"17:25"}}
13:43:07.259 RSL: STATUS10 = {"StatusSNS":{"Time":"2023-11-04T13:43:07"}}
13:43:07.268 RSL: STATUS11 = {"StatusSTS":{"Time":"2023-11-04T13:43:07","Uptime":"0T00:09:50","UptimeSec":590,"Heap":20,"SleepMode":"Dynamic","Sleep":10,"LoadAvg":101,"MqttCount":0,"POWER1":"ON","POWER2":"ON","POWER3":"ON","POWER4":"ON","POWER5":"ON","Dimmer":100,"Color":"FF032200","HSBColor":"353,99,100","White":0,"Channel":[100,1,13,0],"Scheme":0,"Fade":"OFF","Speed":1,"LedTable":"ON","Wifi":{"AP":1,"SSId":"Servus nachad!","BSSId":"3C:37:12:84:15:FB","Channel":1,"Mode":"11n","RSSI":28,"Signal":-86,"LinkCount":1,"Downtime":"0T00:00:03"}}}

```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:
...
13:48:38.920 SRC: WebGui from 192.168.178.74
13:48:38.928 RSL: RESULT = {"POWER5":"ON"}
13:48:38.931 RSL: POWER5 = ON
13:48:39.585 CFG: Saved to flash at F6, Count 190, Bytes 4096

and then the reboot happens

```

### TO REPRODUCE
_Steps to reproduce the behavior: Just play aroud with the sliders and you will see a reboot if the saturation is on 100 percent. I think it happens more often if there is a change from white leds to rgb leds.

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by sfromis** [comment]
Then don't do that. Seriously.

The point is that the hardware is most likely designed up to a specific power level, and overtaxing the power supply can lead to things like lacking voltage, overheating, or potentially fire.

You can use the command `DimmerRange` to config a safe range.

**Reply by stefanbode** [reply]
Agree to sfromis. This is very likely a non sufficient power supply and has nothing to do with TASMOTA.

**Comment by boaschti** [comment]
I couldn't belive that the power supply is the problem. It seems nearly unpossible for me. On this module I have connected 4 Leds a 10mA and 4 ssr a 15mA. The power supply is a HiLink 2.5W and a AM1117 3V3 LDO. 
Yesterday i flashed the same fw to a Wemos d1 without any load an could not cause the error.
Very strange.
